qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/sections/BaseWhenSection.java

```diff
@@ -100,4 +100,9 @@ public static boolean shouldCompleteWhenSection(Section section) {
 		}
 		return true;
 	}
+	
+	@Override
+	public boolean isValidOperator(String partName) {

```

Comment:
Sorry I forget to mention that operators (isValidOperator) should be moved in a new class BaseCaseSection which is extended by CaseSction and IsSection.

indeed it isoperator for #case and not for #when.